mandatory/4-tax.js

```diff
@@ -17,7 +20,10 @@ function calculateSalesTax() {}
   Remember that the prices must include the sales tax (hint: you already wrote a function for this!)
 */
 
-function addTaxAndFormatCurrency() {}
+function addTaxAndFormatCurrency(productPrice) {
+  let total = calculateSalesTax(productPrice);
+  return "£" + total.toFixed(2);
+}
 
 /* 
 ===================================================
```

Comment:
Had a great time going through your codes. it gave me alternate perspectives to solve a problem! 

Thumbs up Farzad!

Reply:
Thanks a lot Chioma